xbot_service_interface/examples/echo_with_schema.py

```
@@ -66,6 +66,14 @@ def got_count(value: int, timestamp: int):
                 msg = f"Echo request {i}"
                 print(f"Sending: {msg!r}")
                 echo.send_input_text(msg)
+
+                if i % 5 == 0:
+                    try:
+                        result = echo.call_rpc_echo_test("hi", 30, timeout_ms=1000)
+                        print(f"  RpcEchoTest result: {result!r}")
+                    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)